### ISSUE TITLE
Add optional chemotherapy dose intensity fields

### ISSUE THREAD

**Issue by hknahal** [icgc-argo/argo-dictionary#323]
Youngil Koh (clinician and member of TCA WG) said they are collecting treatment discontinuation and dose-intensity information and plan to extract toxicity information causing dose reduction or treatment discontinuation. During our discussions, he recommended the following guide. Instead of requesting the exact percentage of dose intensity reduction, a range is sufficient (easier to analyzed later). He explained either a dose delay or dose delay + dose reduction can happen, so it's better not to divide them and to unify them as "reduced dose intensity".

Proposed optional clinical fields to be added to Chemotherapy table:

Field name: reduced_dose_intensity
Description: Was there a significant reduction in dose intensity? 
Accepted values: (Yes/No)

Field name: dose_intensity_reduction_form
Description: If Yes, what form of dose reduction intensity?
Accepted values: ['Dose reduction', 'Dose delay or dose omission', ' Both']
Validation required: This field will be required if 'reduced_dose_intensity' as 'Yes'

Field name: percentage_dose_intensity_reduction
Description: If Yes, indicate how much the dose was changed
Accepted values: ['20-49%', '>= 50%']
Validation required: This field will be required if 'reduced_dose_intensity' is submitted as 'Yes'

**Comment by hknahal** [comment]
Clarification required about whether amount of dose intensity reduction applies to dose delay

Followed up with Youngil who explained the following: 

My email:
```
For question #3 it is asking for the amount reduced (as a percentage). My understanding is that question #3 should only apply if question #2 is submitted as "1) Dose reduction" or "3) Both". My understanding of "Dose delay" means the drug dose is not reduced but is delayed by a certain number of days, and "Dose omission" means the drug dose is reduced to 0.  So for this reason, the percentage of how much the dose was reduced should not apply if Question #2 is submitted as "2) Dose delay or dose omission".  Is this correct?
```
Youngil's response:
```I think decreased dose-intensity is calculable if there is dose delay. (e.g. q 3 weeks schedule --> q 4 weeks schedule: 25% DR) Less than 25% would be also there to catch exact data.```

**Action Items:**
1. Will add another term ">20%" to the "percentage_dose_intensity_reduction" field as per Youngil's recommendation.
2. Will add validation check which requires "dose_intensity_reduction_form" if "reduced_dose_intensity" is "Yes"

**Comment by kcullion** [comment]
@hknahal I tested this on QA and it looks good to me! I uploaded the files that I used here: https://drive.google.com/drive/folders/1wl84B9EJcdcYqoZzOoe4hJxoPYabKR22?usp=sharing

I tested the following: 
1. If fields are optional (working)
2. If dose_intensity_reduction = yes, I need to fill out the other two fields (working)
3.  If dose_intensity_reduction = no, and I fill out the other two fields, I get an error (working)
4. saved from excel to see if % character is ok - when I convert cells to text format, then save the excel as tsv it still works!

Errors eg:
<img width="1136" alt="dictionary323-chemo-invalid-excel" src="https://user-images.githubusercontent.com/18424138/140100492-51f3975a-79a2-4e7f-8d90-7b50d6676f15.png">
<img width="1414" alt="dictionary323-chemo-invalid" src="https://user-images.githubusercontent.com/18424138/140100499-5011c000-124f-4606-b2fb-b66f862881ad.png">
